### ISSUE TITLE
x11-base/xwayland-21.1.1-r1 crashes with -fipa-pta under sway

### ISSUE THREAD

**Issue by B4ckslash** [InBetweenNames/gentooLTO#766]
First, some info:

```
GCC version: 11.1.0-r1
Glibc version: 2.33-r1
ACCEPT_KEYWORDS: "amd64 ~amd64"
```

**TL;DR: xwayland crashes with -fipa-pta (breaking the X connections of other applications) when querying GLX info**

After the latest round of updates, I encountered a strange issue: some XWayland applications would only work with specific flags (e.g. spotify with --disable-gpu) and certain XWayland applications would crash the whole embedded X server, resulting other applications not having a window anymore but still running (i.e. there still was music playing from Spotify etc.). This was the error they logged:
`X connection to :1 broken (explicit kill or server shutdown)`
The worst offender of crashing others was Lutris, and the other applications then logged that the X connection to the display was broken.

Turns out, this happened because Lutris queried for GLX information, and could be reproduced by simply running glxinfo in a terminal. After a lot of diagnosis, I figured out that XWayland crashed when trying to get display info. After trying various versions of Sway, Wlroots, and Mesa, I managed to figure out that building XWayland without -fipa-pta resolved the issue, which is why I suggest to add it to the LTOworkarounds.

**Comment by B4ckslash** [comment]
Also: I know that the description is a bit more verbose than perhaps necessary, but I wanted to make sure that others can find this issue if they have similar problems. I nearly went mad trying to diagnose the cause of this behavior and Google didn't turn up anything usable for the errors I got.

**Comment by InBetweenNames** [comment]
Thanks for investigating!  I'll accept your PR as soon as it passes the review.
